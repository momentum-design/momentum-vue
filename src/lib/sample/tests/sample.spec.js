import { shallowMount } from '@vue/test-utils';
import Sample from '../sample.vue';

describe('Sample', () => {
  it('should match snapshot', () => {
    const wrapper = shallowMount(Sample);
    expect(wrapper.html()).toMatchSnapshot();
  });
});
