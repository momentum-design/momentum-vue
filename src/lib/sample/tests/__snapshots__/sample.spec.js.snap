// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Sample should match snapshot 1`] = `<main class="md-sample"></main>`;
